docs/upgrade/UPGRADE-v8.1.0-dev.md

```diff
@@ -125,7 +125,22 @@ There you can find links to upgrade notes for other versions too.
         + docker-compose exec -T php-fpm composer install
         + docker-compose exec -T php-fpm ./phing db-create test-db-create build-demo-dev-quick error-pages-generate
         ```
-
+- Run `php phing fix-annotations` to fix or add all the relevant annotations for your extended classes ([#1344](https://github.com/shopsys/shopsys/pull/1344))
+    - Thanks to the fixes, your IDE (PHPStorm) will understand your code better.
+    - You can read more about the whole topic in the ["Framework extensibility" article](../introduction/framework-extensibility.md#making-the-static-analysis-understand-the-extended-code).
+    - The checks and fixes of the proper annotations are now included in all `standards-*` phing targets, if you want to turn this feature off, add the following line into your `build.xml`:
+    ```diff
+  + <property name="check-and-fix-annotations" value="false"/>
```

Comment:
the line has to be added before the import statement and this is not clear from this instruction

Reply:
fixed in 9126a2c7d

Reply:
shit, wrong place :man_facepalming: reverted in 964c0d7d2 and fixed in 4d94cbe50 :smile:

Reply:
omg, fixup of fixup of fixup in 277c5ff3a

Reply:
😄  okay